knip.json

```diff
@@ -0,0 +1,493 @@
+{
+  "$schema": "https://unpkg.com/knip@5/schema.json",
+  "workspaces": {
+    ".": {
+      "entry": [
+        "gulpfile.js"
+      ]
+    },
+    "cli": {
```

Comment:
These entrypoints do seem like they'll be a bit unreasonable to keep up with. Probably a sign of our own monorepo not being so well organized. Maybe esm migration will help with this. You can kind of tell which packages are most unruly just by the amount of rules or ignores needed.